app/views/pages/terms_and_conditions.html.erb

```diff
@@ -0,0 +1,210 @@
+            </p>
+            <p>
+                Once the first user has been onboarded to a school, this user can add subsequent users to the school account.
+            </p>
+            <p>
+                Users must only allow someone working for their school to publish or manage school experience listings for their school.
+            </p>
+            <p>
+                Users must not allow third parties to publish or manage school experience dates on behalf of their school.
+            </p>
+            <p>
+                If the Get School Experience team detects, or is alerted to, any usage of an account by third parties that user account will be deleted immediately and the user will no longer have access to publish or manage school experience dates for that school.
+            </p>
+        <h3 class="govuk-heading-s">Unacceptable use</h3>
+            <h4 class="govuk-heading-xs">School experience listings</h4>
+            <p>
+                You must not publish, edit or delete a school experience listing if you do not have authority from the school, trust or local authority to do so.
+            </p>
+            <p>
+                You must not publish:
+            </p>
+                <ul class="govuk-list govuk-list--bullet">
+                    <li>offensive or inappropriate language, including but not limited to profanities or language pejorative of any group, such as racist, sexist or homophobic comments</li>
+                    <li>discriminatory content: you must not state or imply in a school experience listing that you’ll discriminate against anyone (subject to certain exceptions under the Equality Act 2010, particularly those applying to schools with a religious character); for further information see GOV.UK's <%= link_to 'guidance on discrimination in job adverts (opens in a new tab)', 'https://www.gov.uk/employer-preventing-discrimination/recruitment', target: "_blank" %></li>
+                    <li>material that infringes any intellectual property rights, such as copyright and trademarks (this means generally that you must own the rights in everything you submit or must obtain permission from the rights owner to submit the material)</li>
+                    <li>material that's technically harmful (including, but not limited to, computer viruses, logic bombs, Trojans, worms, harmful components, corrupted data or other malicious software or harmful data)</li>
+                    <li>material that encourages or teaches conduct that's a criminal offence, gives rise to civil liability, or is otherwise unlawful</li>
+                </ul>
+        <h3 class="govuk-heading-s">Accepting applications through the Get school experience service</h3>
+            <h4 class="govuk-heading-xs">By accepting school experience applications through the Service, you agree that:</h4>
```

Comment:
@gemmadallmandfe accessibility question here. This is styled as a header in the draft, but I'm not convinced it should be. Agree?

Reply:
Do you mean the h4 `By accepting.....`?

If so, agree that this should not be a heading (looks like you've already changed it)